Appraisals

```diff
@@ -1,5 +1,5 @@
 appraise "sass-3-4" do
-  gem "sass", "~> 3.4"
+  gem "sass", "~> 3.4.22"
```

Comment:
Ooh, I hadn't thought of that. What do you think would be a better way of specifying it?